app/build.gradle

```diff
@@ -90,6 +90,7 @@ dependencies {
   kaptAndroidTest(
       'com.google.dagger:dagger-compiler:2.24'
   )
+  api project(':data')
```

Comment:
Had to introduce this because of error in access to retrofit
Reference link for solution: https://stackoverflow.com/questions/47124987/dagger-2-cannot-access-retrofit

Reply:
Does this need to be ``api``? If so, it's a good place to put a TODO on #59 to remove this once we can use Bazel, since ``api`` is deprecated in Gradle and so this solution may eventually break (whereas with Bazel it shouldn't).

Reply:
Yes we need this `api`. Added TODO accordingly.